.github/workflows/ci.yml

```diff
@@ -38,7 +38,7 @@ jobs:
       if: runner.os == 'macOS'
     - name: Install elan
       run: |
-          curl https://raw.githubusercontent.com/Kha/elan/master/elan-init.sh -sSf | sh -s -- --default-toolchain leanprover-community/lean:3.5.1 -y 
+          curl https://raw.githubusercontent.com/leanprover/elan/master/elan-init.sh -sSf | sh -s -- --default-toolchain leanprover-community/lean:3.5.1 -y 
```

Comment:
This hardcodes Lean 3.5.1; I guess it hasn't hurt up to now, but this might be something we want to change.

Reply:
```suggestion
          curl https://raw.githubusercontent.com/leanprover/elan/master/elan-init.sh -sSf | sh -s -- --default-toolchain none -y
```